libHVCAENx527App/src/CAENHVWrapper_sim.cpp

```diff
@@ -11,6 +11,9 @@
 #define NUM_CH 2
 #define BOARDNAME "s1535s"
 
+static unsigned g_call_out = 0;
+static unsigned g_error_on_call = 1000;
```

Comment:
I'm curious about the choice of unsigned types here. It's not necessarily wrong but seems to be discouraged in modern c/c++ guidelines.

Reply:
It is an incrementing counter and I chose unsigned as I specifically wanted the wrap around behaviour and not to worry about potentially undefined behaviour if the integer overflows. That is probably unlikely given the IOC lifetime though. The c++ guidelines i believe are "do no use unsigned just because it should not be negative" as this can give performance issues and also issues when signed and unsigned are compared or assigned.